### PR TITLE
Add steps to save address when using faucet

### LINKED ISSUE #105: Faucet Guide lacks important steps - saving vanity address
The [Guide ](https://guide.kusama.network/en/latest/start/faucet/) is good and written so that even users new to Kusama can get onboarded easily.
However it lacks a step between 1) Generate an address and 2) Submit an issue. This is 1.2) Saving vanity address with private keys etc.
Furthermore a step 3) transferring ksm from Faucet address to initial address would be helpful as well.

Seems intuitive, but there are users (including myself) who do not understand the vanity address logic and therefor don´t receive the ksm they expect to get. The guide which is in itself dummy-proven, leaves this part out. Would be great, if it could be edited.

### PR DESCRIPTION
**Pull request by laboon**
Fixes #105

We did not have steps to save the generated vanity address when using the faucet, leading several people to generate an account, send tokens to it, but no longer have access to the address (since it disappears after the page is closed) . I have added the additional steps along with images.

@chevdor can you review please, since you're already aware of the issue?

### CODE REVIEW

File: docs/start/faucet.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ In order to prevent Sybil attacks, the faucet will only drip to users that have
 
 The faucet will only drip to addresses that contain the string `ksma`(case-insensitive, that is, `KSMA` or `kSmA` would work). It will also never drip to the same address twice. Thus, a new address has to be created each time a user requests KSM tokens.
 
-Obtaining a valid address will require generating numerous addresses until you find one which meets the requirements. You can think of this as a small proof-of-work. You can use the command-line tool [Subkey](#Using-Subkey), the web interface ([PolkadotJS Dashboard](#Using-PolkadotJS-Dashboard)), or any other program capable of generating arbitrary addresses.  You can even code up your own if you like.
+Obtaining a valid address will require generating numerous addresses until you find one which meets the requirements. You can think of this as a small proof-of-work. You can use the command-line tool [subkey](#Using-Subkey), the web interface ([PolkadotJS Dashboard](#Using-PolkadotJS-Dashboard)), or any other program capable of generating arbitrary addresses.  You can even code up your own if you like.
```
Comment (chevdor) [inline docs/start/faucet.md]: I would mention:

> which meets the requirements...

of the faucet.

File: docs/start/faucet.md [hunk 2]
```diff
@@ -24,11 +24,27 @@ Obtaining a valid address will require generating numerous addresses until you f
 2. Under the [Accounts page](https://polkadot.js.org/apps/#/accounts), select `Vanity Address` tab.
 ![](https://i.imgur.com/EZw5iOx.png)
 
-3. Input `ksma` in the "search for" textbox and leave the "keypair crypto type" by default unless you want to use another type.  After this, click `Start generation`. It will most likely take at least 10 mins to generate the address pattern you want.
+3. Input `ksma` in the "search for" textbox and leave the "keypair crypto type" by default unless you want to use another type.  After this, click `Start generation`. It may take 10 minutes or more to generate the address pattern you want, depending on the power of your hardware.
```
Comment (chevdor) [inline docs/start/faucet.md]: I would add:

> You may leave YES in the case sensitive field but switching to NO will greatly speed up the process.

File: docs/start/faucet.md [hunk 3]
```diff
@@ -24,11 +24,27 @@ Obtaining a valid address will require generating numerous addresses until you f
...
 ![](https://i.imgur.com/ckbunwK.png)
 
+4. After you see that a valid `ksma` address has been created, press the blue `+` button to generate an account.
```
Comment (chevdor) [inline docs/start/faucet.md]: I would add:

> (Note that the case does not matter, any of the following are ok: kSmA, KSMA, ksMA, etc...)

File: docs/start/faucet.md [hunk 4]
```diff
@@ -84,6 +100,12 @@ Once you have generated your Kusama address containing the string `ksma`, you ar
 6. You will see a comment posted to your issue indicating success or failure.  Success means that 0.1 KSM has been sent to the posted address; failure means that there was some problem and no KSM was sent to your address. In the event of failure, the message should indicate what the problem is (e.g., the faucet is dry, the address is invalid).
 7.  The issue will close automatically after you receive a response, whether or not it was a success.
 
+### After
+
+Remember that you must drip to a different address each time - you will not be able to get a drip from the faucet to this address again.  You also may have a more secure way of storing addresses such as cold wallet, or simply keep all of your KSM at one address.
+
+Therefore, you may wish to move your KSM to a different address after this.  You should be able to move your KSM immediately after receiving them.
+
```
Comment (chevdor) [inline docs/start/faucet.md]: I would add a section with a title similar to "So I got KSM, now what?"

I would mention that it is NOT possible to transfer 0.1KSM to another account as the added fee will add up over 0.1 KSM. Transferring 0.085 KSM should work most of the time (to be checked).

Owning KSM gives you super powers. Among them you may:
- obviously you can transfer them to another account
- nominate validators and passively EARN more KSM
- support (=second) democratic proposals
- vote for or against active referenda
- vote and support one or more of your favorite council members
- set your on-chain identity
- and way more!

**Comment by chevdor** [comment]
@vfreudenhammer do you think this change would have prevented the issue you ran into or do you see other point to clear up?

File: docs/start/faucet.md (hunk 2 above)
Comment (laboon) [reply docs/start/faucet.md]: Sadly, I missed this step myself...

File: docs/start/faucet.md (hunk 3 above)
Comment (laboon) [reply docs/start/faucet.md]: added

**Comment by laboon** [comment]
@chevdor do you want to double-check my additions?

**Comment by chevdor** [comment]
LGTM I just would like to confirm the 0.08 KSM value.
Let me run a test, I will confirm here.

**Comment by chevdor** [comment]
Ok, in my one test, transferring 0.080 KSM did work and 'apparently' wiped the sender account as it brought under the existential deposit balance.
https://polkadot.js.org/apps/#/explorer/query/0x32991ab440038600eed19bfc8cb4ec3547c216b9b56ce166f05e19812e0b7208

So I would leave 0.080 KSM. If users reports issue, we may be more conservative and go down to 0.075 KSM but I think it should be ok for now.

**Pull request merged**